src/js/components/RadioButton/__tests__/RadioButton-test.js

```diff
@@ -123,4 +124,38 @@ describe('RadioButton', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  describe('check size', () => {
+    test('default value', () => {
+      const { container } = render(
+        <Grommet theme={grommet}>
+          <RadioButton name="test" checked onChange={jest.fn()} />
+        </Grommet>,
+      );
+
+      expect(
+        container.querySelector('circle').hasAttribute('r', '6'),
+      ).toBeTruthy();
+    });
+
+    test('custom value', () => {
+      const customTheme = {
+        radioButton: {
+          check: {
+            size: '8',
+          },
+        },
+      };
+
+      const { container } = render(
+        <Grommet theme={customTheme}>
+          <RadioButton name="test" checked onChange={jest.fn()} />
+        </Grommet>,
+      );
+
+      expect(
+        container.querySelector('circle').hasAttribute('r', '6'),
```

Comment:
On this second test should this be 8 instead of 6 since the check size is being set to 8 in the customTheme?
```
        container.querySelector('circle').hasAttribute('r', '8'),
```

Reply:
Good catch, also this should be failing because of that. Thanks

Reply:
Sounds fishy that the test passed when it was size, what am I missing?